src/eds_files/Arduino1.eds

```diff
@@ -1211,7 +1211,7 @@ SupportedObjects=2
 [2000]
 ParameterName=Distance
 ObjectType=0x7
-DataType=0x0005
+DataType=0x0003
 AccessType=rw
-DefaultValue=0
+DefaultValue=1023
```

Comment:
focus je trouwens nog niet heel erg op de .eds van de arduino's. Ik ben er nog heel erg aan het experimenteren hiermee

Reply:
Ja dat is goed ik had het alleen maar aangepast om te kijken of die value ook goed ging, eerst ging het niet goed :) Nu wel gelukkig